### DISCUSSION TITLE
【求助】11号伤害上不去，怎么办啊

### DISCUSSION THREAD

**Discussion by mhpmc** [share121/inter-knot#566] [其他]
![bb8dbd90eb3d79a2c4be6f92d493ec67255839852](https://github.com/user-attachments/assets/539cb792-bb3f-4188-970d-1b8937e51607)
11号sheny深渊刮不动一点，有没有大佬推荐一下11号配队，4星的

**Comment by FrostPhoenix79** [comment]
派派加露西，打先手异常锤。
能刮动，才不是我没有五星角色呢（呜呜真的没有五星队友）
